Configuration/PyReleaseValidation/scripts/runTheMatrix.py

```diff
@@ -87,7 +87,8 @@ def runSelected(opt):
                      10024.0, #2017 ttbar
                      10224.0, #2017 ttbar PU
                      10824.0, #2018 ttbar
-                     11634.911, #2021 DD4hep ttbar
+                     11634.911, #2021 DD4hep ttbar reading geometry from XML
+                     11634.912, #2021 DD4hep ttbar reading geometry from the DB
```

Comment:
This should also be disable because short matrix will run .912 also if it is here.

Reply:
@srimanob @chayanit 912 is commented out now in the short matrix.

Reply:
Thanks @cvuosalo